relay-cabi/include/relay.h

```diff
@@ -97,6 +96,10 @@ enum RelayDataCategory
    * TODO(jferg): Rename this to UserFeedback once old UserReport is deprecated.
    */
   RELAY_DATA_CATEGORY_USER_REPORT_V2 = 14,
+  /**
+   * Metric bucket.
+   */
+  RELAY_DATA_CATEGORY_METRIC_BUCKET = 15,
```

Comment:
Note that this will require a release of the python library and bump in Sentry and Snuba.